### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by ekarlso**
Terraform 1.8.x
Terraform provider 0.48.0 (Remove deprecated resources)
Crossplane deps

### Description of your changes

Fixes #

I have:

- [x] Read and followed Crossplane's [contribution process].
- [x] Run `make reviewable test` to ensure this PR is ready for review.

### How has this code been tested

[contribution process]: https://git.io/fj2m9

### CODE REVIEW

File: config/me/config.go [hunk 1]
```diff
@@ -19,10 +19,6 @@ func Configure(p *config.Provider) {
 			Type: "github.com/edixos/provider-ovh/apis/me/v1alpha1.Group",
 		}
 	})
-	p.AddResourceConfigurator("ovh_me_ssh_key", func(r *config.Resource) {
-		r.ShortGroup = shortGroup
-		r.Kind = "SSHKey"
-	})
```
Comment (smileisak) [inline config/me/config.go:25]: @ekarlso is there any reason to delete this ResourceConfigurator?

**Review by smileisak — CHANGES_REQUESTED** [review]
Hi @ekarlso,

Thanks again for your PR! :) However, I noticed that no new resources have been added or updated in the config folder.

When adding new resources to the OVH Terraform provider, they should be included in the config folder and referenced in the external_name.go file.

The following new resources need to be configured to be generated as CRDs with their corresponding controller logic:
New Resource: ovh_cloud_project_gateway_interface
([PR #697](https://github.com/ovh/terraform-provider-ovh/pull/697))

New Resource: ovh_cloud_project_region_loadbalancer_log_subscription
([PR #657](https://github.com/ovh/terraform-provider-ovh/pull/657))

New Resource: ovh_dbaas_logs_token
([PR #649](https://github.com/ovh/terraform-provider-ovh/pull/649))

New Resource: ovh_iploadbalancing_udp_frontend
([PR #587](https://github.com/ovh/terraform-provider-ovh/pull/587))

New Resource: ovh_domain_zone_dnssec
([PR #589](https://github.com/ovh/terraform-provider-ovh/pull/589))

New Resource: ovh_vps
([PR #594](https://github.com/ovh/terraform-provider-ovh/pull/594))

New Resource: ovh_ip_firewall
([PR #596](https://github.com/ovh/terraform-provider-ovh/pull/596))

New Resource: ovh_ip_firewall_rule
([PR #601](https://github.com/ovh/terraform-provider-ovh/pull/601))

New Resource: ovh_ip_mitigation
([PR #604](https://github.com/ovh/terraform-provider-ovh/pull/604))

Please ensure these resources are added and updated accordingly.

File: config/me/config.go (hunk 1 above)
Comment (ekarlso) [reply config/me/config.go:25]: Yes, SSHKey has been deprecated from the updated terraform providers but for some reason referenced still.

**Comment by ekarlso** [comment]
Hi, so I am excluding the ovh_vps resource as it causes issues as described here: https://github.com/crossplane/upjet/issues/372#issuecomment-2342792837

**Comment by ekarlso** [comment]
I forgot that the change is pending https://github.com/crossplane/upjet/pull/437  to get merged first. I've pinged them in their pull.

**Comment by smileisak** [comment]
I have fixed dependencies, rolled back to main's go.mod version.

**Pull request merged**